src/style.css

```diff
@@ -1 +1,63 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  margin: 0;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: Avenir;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.search {
+  font-weight: 300;
+  background-image: url(images/Icon.png);
+  background-repeat: no-repeat;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+}
+
+.search::placeholder {
+  color: #3d4e61;
+}
+
+.search:hover {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.search:focus {
+  font-weight: bold;
+}
+
+.search--big {
+  height: 70px;
+  background-position: 26px 25px;
+  background-size: 19px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.search--small {
+  height: 42px;
+  background-position: 13px 15px;
+  background-size: 11px;
+  padding-left: 33px;
+  font-size: 14px;
+}
```

Comment:
You are setting the height of the search box. This is not necessary unless there is a specific requirement to do so. Let the content size dictate it.